packages/codegen-ui-react/lib/imports/import-collection.ts

```diff
@@ -109,6 +109,19 @@ export class ImportCollection {
     ];
   }
 
+  getAliasMap(): { model: { [modelName: string]: string } } {
+    const modelMap: { [modelName: string]: string } = {};
+    const modelSet = this.#collection.get(ImportSource.LOCAL_MODELS);
+    const modelAliases = this.importAlias.get(ImportSource.LOCAL_MODELS);
+    if (modelSet) {
+      [...modelSet].forEach((item) => {
+        const alias = modelAliases?.get(item);
+        if (alias) modelMap[item] = alias;
+      });
+    }
+    return { model: modelMap };
```

Comment:
Why not just return `modelMap`?

Reply:
I wanted to make it extensible in case we wanted to return other aliases, such as that of custom Figma components.